app/models/upload.rb

```diff
@@ -34,6 +34,7 @@ def access_control_post
   attr_accessor :for_export
   attr_accessor :for_site_setting
   attr_accessor :for_gravatar
+  attr_accessor :validate_file_size
```

Comment:
Can we use this here?

```suggestion
  attr_accessor_with_default :validate_file_size, true
```

Reply:
From what I see it was a thing in Rails 2.x and eventually was deprecated and removed in 3.x